app/lib/settings/pages/settings.dart

```diff
@@ -7,17 +7,48 @@ class SettingsPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ListView(children: [
-      Card(
-        child: ListTile(
-          leading: Icon(Icons.delete),
-          title: Text(context.l10n.settings_page_delete_data),
-          trailing: Icon(Icons.chevron_right),
-          onTap: () => showDialog(
-            context: context,
-            builder: (_) => _deleteDataDialog(context),
-          ),
+      ListTile(
+        title: Text(
+          context.l10n.settings_page_account_settings_headline,
+          style: PharmeTheme.textTheme.bodyLarge,
+        ),
+      ),
+      ListTile(
+        leading: Icon(Icons.delete),
+        title: Text(context.l10n.settings_page_delete_data),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => showDialog(
+          context: context,
+          builder: (_) => _deleteDataDialog(context),
+        ),
+      ),
+      Divider(),
+      ListTile(
+        title: Text(
+          context.l10n.settings_page_more_headline,
+          style: PharmeTheme.textTheme.bodyLarge,
         ),
       ),
+      ListTile(
+        title: Text(context.l10n.settings_page_onboarding),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => context.router.push(OnboardingRouter()),
+      ),
```

Comment:
This kind of breaks the app flow. I think we should not offer this button.

Reply:
The button was included in the mockups and I do actually see why it could be helpful. Otherwise there wouldn't be any opportunity do get the information from the onboarding again or just see it again for other reasons. And without having to log in again after the onboarding, I don't see a reason to remove it (for me it doesn't break the flow).
@LeonHermann322 do you have an opinion on that?

Reply:
I would leave it in.